### PR TITLE
chore(documentation): meta tags

### PR DESCRIPTION
**Pull request by rachelbt**
preview image is needed

### CODE REVIEW

File: apps/docs/_layouts/base.njk [hunk 1]
```diff
@@ -5,6 +5,19 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!--  Essential META Tags -->
+    <meta property="og:title" content="Vivid Web Components">
+    <meta property="og:type" content="Documentation" />
+    <meta property="og:image" content="../../../vivid-logo.svg">
```
[the remark this answers is not in the document]
Comment (jshenkman) [reply apps/docs/_layouts/base.njk:11]: > @jshenkman @AyalaBu should we have a dedicated image for this social sharing banner?

nope, but I'll make one! 
What do you want it to say? Vivid web components? @yinonov

[the remark this answers is not in the document]
Comment (yinonov) [reply apps/docs/_layouts/base.njk:11]: just some nice image, no text. we'll add the text

**Review by yinonov — APPROVED** [review]

File: apps/docs/_layouts/base.njk [hunk 2]
```diff
@@ -8,7 +8,7 @@
     <!--  Essential META Tags -->
     <meta property="og:title" content="Vivid Web Components">
     <meta property="og:type" content="Documentation" />
-    <meta property="og:image" content="../../../vivid-logo.svg">
+    <meta property="og:image" content="../../../vivid-logo.png">
```
Comment (yinonov) [inline apps/docs/_layouts/base.njk:11]: reduce file size
```suggestion
    <meta property="og:image" content="../../../vivid-logo.jpg">
```

Comment (yinonov) [reply apps/docs/_layouts/base.njk:11]: I also think it should be full url and not a relative path

**Review by yinonov — CHANGES_REQUESTED** [review]

File: apps/docs/_layouts/base.njk (hunk 2 above)
Comment (rachelbt) [reply apps/docs/_layouts/base.njk]: I will check it once the deploy will work :)

**Review by yinonov — APPROVED** [review]

**Pull request merged**